core/src/verify.rs

```diff
@@ -454,6 +487,22 @@ impl CommitSequenceVerifier {
         self.total_commits.push(commit.clone());
         Ok(())
     }
+
+    /// To verify that the newly added previous_block_hash field effectively prevents replay attacks
+    /// Check the integrity of the chain of agendas when processing new agendas
```

Comment:
I think this comment is not that perfect and redundant with the comment in apply_commit. How about just deleting it here?

Reply:
reflected